feat(dataarts): add new datasource for query security permission sets (#9413)
huaweicloud/terraform-provider-huaweicloud: docs/data-sources/dataarts_security_permission_sets.md

---
subcategory: "DataArts Studio"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_dataarts_security_permission_sets"
description: |-
  Use this data source to get the list of permission sets for DataArts Studio Security within HuaweiCloud.
---

# huaweicloud_dataarts_security_permission_sets

Use this data source to get the list of permission sets for DataArts Studio Security within HuaweiCloud.

## Example Usage

### Basic Usage

```hcl
variable "workspace_id" {}

data "huaweicloud_dataarts_security_permission_sets" "test" {
  workspace_id = var.workspace_id
}
```

### Query permission sets by parent ID

```hcl
variable "workspace_id" {}
variable "parent_id" {}

data "huaweicloud_dataarts_security_permission_sets" "test" {
  workspace_id = var.workspace_id
  parent_id    = var.parent_id
}
```

## Argument Reference

The following arguments are supported:

> * `region` - (Optional, String) Specifies the region where the DataArts Studio instance is located.  

Reviewer: ```suggestion
* `region` - (Optional, String) Specifies the region where the permission sets are located.  
```

Author: done
